### DISCUSSION TITLE
Remove organized flag in bulk or from all scenes

### DISCUSSION THREAD

**Discussion by RoachCoach521** [stashapp/stash#2669] [Q&A] [answered]
Is there any way to do a mass removal/disable of the organize flag? I set it early on without realizing it wouldn't allow metadata to be updated, so no unless I do manual pulls on each scene, they never update with tags, etc.

Thanks all.

**Comment by MrX292** [comment, marked as answer]
Filter for Organised is true, change number of shown scenes, click on ... select all, click edit and deselect Organised and hit apply

**Comment by RoachCoach521** [comment]
AHHHHH, you're a lifesaver! I didn't realize the edit function worked this way. Game changer.

Thank you @MrX292
